Library/Homebrew/formula_installer.rb

```diff
@@ -488,6 +494,19 @@ def expand_dependencies(deps)
       end
     end
 
+    if poured_bottle
+      bottle_deps = dependencies_to_pour_bottles
+      bottle_deps = bottle_deps.reject do |dep|
+        options = inherited_options[dep.name] = inherited_options_for(dep)
+        begin
+          dep.satisfied? options
+        rescue FormulaUnavailableError
+          true
```

Comment:
Doesn't seem right to be silently ignoring missing deps here.

Reply:
Fixed.